app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -126,10 +135,17 @@ protected void onCreate(Bundle savedInstanceState) {
 
         viewBinding = OnlinefeedviewActivityBinding.inflate(getLayoutInflater());
         setContentView(viewBinding.getRoot());
-
         viewBinding.transparentBackground.setOnClickListener(v -> finish());
         viewBinding.card.setOnClickListener(null);
         viewBinding.card.setCardBackgroundColor(ThemeUtils.getColorFromAttr(this, R.attr.colorSurface));
+        headerBinding = OnlinefeedviewHeaderBinding.bind(
+                View.inflate(this, R.layout.onlinefeedview_header, null));
```

Comment:
Please use `OnlinefeedviewHeaderBinding.inflate(getLayoutInflater());`

Reply:
It doesn't work and the UI is not accessible when i used that. Should i still do changes?